src/pages/[platform]/build-a-backend/storage/storage-v5-to-v6-migration-guide/index.mdx

```diff
@@ -19,31 +19,31 @@ export function getStaticProps(context) {
   };
 }
 
-This guide will help you migrate v5 Storage implementations to the new functional APIs in v6.
+This guide will help you migrate Amplify JavaScript v5's Storage APIs to the new Amplify JavaScript v6's Storage APIs.
 
 ## Deprecated Options
 
-<Callout>Many input and output properties have been deprecated due to low usage or added complexity. If you would like to see these options re-implemented in v6, feel free to open a [feature request](https://github.com/aws-amplify/amplify-js/issues/new?assignees=&labels=pending-triage&projects=&template=2.feature_request.yaml) in our github repository.</Callout>
+<Callout>Many input and output properties have been deprecated in favor of alternatives with a better developer experience. If you have identified a use case that you feel isn't covered in this migration guide, feel free to open a [GitHub issue](https://github.com/aws-amplify/amplify-js/issues/new?assignees=&labels=pending-triage&projects=&template=2.feature_request.yaml).</Callout>
```

Comment:
Small nit: I wonder if we should just say "please open a" vs "feel free to open a". The latter can sometimes come off a little sarcastic depending on interpretation?

Reply:
adddressed.